src/ontology/mondo-ingest.Makefile

```diff
@@ -201,7 +201,7 @@ $(REPORTDIR)/%_mapping_status.tsv $(REPORTDIR)/%_unmapped_terms.tsv: $(REPORTDIR
 
 components/%-unmapped.owl: components/%.owl reports/%_unmapped_terms.tsv
 	cut -f 1 reports/$*_unmapped_terms.tsv | tail -n +2 > reports/$*_unmapped_terms.txt
-	$(ROBOT) filter -i components/$*.owl -T reports/$*_unmapped_terms.tsv -o $@
+	$(ROBOT) filter -i components/$*.owl -T reports/$*_unmapped_terms.txt -o $@
```

Comment:
@matentzn This was the bugfix. -_-. I was filtering on the `.tsv` and not the `.txt`. I checked the DOID output and the obsolete terms I checked for are gone.

I reran for all ontologies, but only DOID, NCIT, and OMIM are updated. I'm sort of confused by that and wondering if I should be concerned. I can do a little analysis but first--do you have any thoughts?

Reply:
Nope, sounds right to me! Only these have deprecated terms!